src/main/java/taskmaster/tasks/Task.java

```diff
@@ -0,0 +1,55 @@
+package taskmaster.tasks;
+public class Task {
+    /**
+     * Description of task.
+     */
+    protected String description;
+    /**
+     * Boolean on whether the task is done.
+     */
+    protected boolean isDone;
+
+    /**
+     * Constructor for the task class.
+     * Tasks can be set to either done or not done.
+     *
+     * @param description Name of the task.
+     * @param marked String to indicate whether the task is done
+     */
+    public Task(String description, String marked) {
+        this.description = description;
+        if (marked == "marked") {
+            this.isDone = true;
+        } else {
+            this.isDone = false;
+        }
+    }
+    /**
+     * Marks the task as done.
+     */
+    public void markAsDone() {
```

Comment:
Pretty nitpicky, but I believe the Coding Standards say you should write this as `setAsDone` instead.